ai21/clients/common/completion_base.py

```diff
@@ -30,6 +30,7 @@ def create(
         count_penalty: Penalty | NotGiven = NOT_GIVEN,
         epoch: int | NotGiven = NOT_GIVEN,
         logit_bias: Dict[str, float] | NotGiven = NOT_GIVEN,
+        n: int | NotGiven = NOT_GIVEN,
```

Comment:
This is the wrong class. We need the `n` parameter to be under `ai21/clients/studio/resources/chat/chat_completions.py`